features/step_definitions/steps.rb

```diff
@@ -9,3 +9,15 @@
 Given(/^there is a dish named "([^"]*)" in our database$/) do |dish_name|
   Dish.create(name: dish_name)
 end
+
+Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
+  Dish.create(name: dish_price)
+end
+
+Then(/^I should se a message saying "([^"]*)"$/) do |message|
+  expect(page).to have_content message
+end
+
+Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
+  Dish.create(name: dish_description)
```

Comment:
And here the dish name would be whatever you pass in as description.

Reply:
Changed to Dish.create(description: dish_description)!